app/lib/account/backend.dart

```diff
@@ -265,4 +267,51 @@ class AccountBackend {
 
     return user;
   }
+
+  /// Creates a new session for the current authenticated user.
```

Comment:
```dart
/// And returns the `sessionId`.
/// 
/// The `sessionId` is a secret that will be stored in a secure cookie.
/// Presence of this sessionId in a cookie, can only be used to authorize user specific content to be embedded
/// in HTML pages (such pages must have `Cache-Control: private`, and may not be cached in server-side). 
/// JSON APIs whether fetching data or updating data cannot be authorized with a cookie carrying the `sessionId`.
```

Maybe this should be written somewhere else... Ideally in the place where HTTP request handlers lookup the current session. Like `requiredAuthenticatedUser` we'll probably need something similar for sessions (though that probably just returns `null` instead of throwing an exception.